core/conn/spj_init/src/main/resources/init_spj.sql

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+# @@@ START COPYRIGHT @@@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# @@@ END COPYRIGHT @@@
+
+SERVER_JAR=${MY_SQROOT}/export/lib/spj_init.jar
+CI_SERVER_LOG=${MY_SQROOT}/ciserver/cislog
+SQLCI=${MY_SQROOT}/export/bin32/sqlci
+CP=/bin/cp
+MANAGEABILITY_CATALOG=TRAFODION
+CIS_SCHEMA=DEFAULT_SPJ
+
+
+function dropAndCreateSchema {
+    echo "Creating Schema for CI Server Component"
+    sqlci << sqlciEOF
+
+      cqd CAT_IGNORE_ALREADY_EXISTS_ERROR 'on';
+      cqd CAT_IGNORE_DOES_NOT_EXIST_ERROR 'on';
+
+      DROP SCHEMA $MANAGEABILITY_CATALOG.$CIS_SCHEMA CASCADE;
+      CREATE SCHEMA $MANAGEABILITY_CATALOG.$CIS_SCHEMA;
+
```

Comment:
This relates to the role comment Hans made below.  We could create a role and change the AUTHORIZATION clause to be the role.  But this is only valid when authorization is enabled.  So you need logic to handle to set the correct authorization clause and code is needed when authorization is enabled and disable to change the schema owner.  I would, for simplicity, make DB__ROOT or DB__ROOTROLE the owner and require the user to set privileges to whatever role or user they want to manage privileges.